cli.py

```diff
@@ -7,12 +7,10 @@
    - personify: Generate the Persona-Chat formatted file.
 """
 import click
+from os import cpu_count
 from preprocessor import preprocess
 from metadataExtractor import getMetadata
 from personifier import personify
-import pandas as pd
-
-df = pd.read_csv("twitter_corpora/twitter/twcs.csv")

```

Comment:
Moved this to the inside of each function, so that the cli.py does not need the panda library